utilities/test_suite/HIP/runTests.py

```diff
@@ -44,7 +44,17 @@ def validate_and_remove_files(path):
         exit()
 
     elif os.path.exists(path):  # check if the folder exists
-        os.system("rm -rvf {}/*".format(path))  # Delete the directory if it exists
+        # Get a list of files and directories within the specified path
+        items = os.listdir(path)
+
+        if items:
+            # The directory is not empty, delete its contents
+            for item in items:
+                item_path = os.path.join(path, item)
+                if os.path.isfile(item_path):
+                    os.remove(item_path)
+                elif os.path.isdir(item_path):
+                    shutil.rmtree(item_path)     # Delete the directory if it exists
```

Comment:
Have we verified the rmtree action for corner cases? item_path is hopefully verified to be within working dir.